### ISSUE TITLE
Evaluate Puppet and Packer for creating Kubernetes cluster

### ISSUE THREAD

**Issue by antonputra** [mapreducelab/kubernetes-ansible#7]
We may need to switch from Ansible to [Puppet](https://puppet.com) or [Hashicorp Packer](https://www.packer.io/intro/index.html). In prod, ansible requires ssh access to each host, since Yubikeys enabled on all hosts, it will be impossible to use it. It will be solved by introducing Vault, ETA for the that project is around 1 year from now.

**Comment by akuksin** [comment]
Let's go without direct provisioning of SW on VMs and rely on [Hashicorp Packer](https://www.packer.io/intro/index.html).

**Comment by antonputra** [comment]
I agree, but the fastest way to get in prod is a puppet(3x version) unfortunate.
